CHANGELOG.md

```diff
@@ -7,11 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
+- [#xx](https://github.com/nf-core/proteinfamilies/pull/xx) - Added nf-test and `meta.yml` file for local subworkflow `GENERATE_FAMILIES`
+  Added nf-test and `meta.yml` file for local subworkflow `REMOVE_REDUNDANCY`
+  Added nf-test and `meta.yml` file for local subworkflow `UPDATE_FAMILIES`
 - [#88](https://github.com/nf-core/proteinfamilies/pull/88) - Added nf-test and `meta.yml` file for local module `BRANCH_HITS_FASTA`
-- [#88](https://github.com/nf-core/proteinfamilies/pull/88) - Added nf-test and `meta.yml` file for local module `FILTER_NON_REDUNDANT_FAMS`
-- [#88](https://github.com/nf-core/proteinfamilies/pull/88) - Added nf-test and `meta.yml` file for local module `IDENTIFY_REDUNDANT_FAMS`
-- [#88](https://github.com/nf-core/proteinfamilies/pull/88) - Added nf-test and `meta.yml` file for local module `EXTRACT_FAMILY_REPS`
-- [#81](https://github.com/nf-core/proteinfamilies/pull/81) - Added the default pipeline end-to-end nf-test.
+  Added nf-test and `meta.yml` file for local module `FILTER_NON_REDUNDANT_FAMS`
+  Added nf-test and `meta.yml` file for local module `IDENTIFY_REDUNDANT_FAMS`
+  Added nf-test and `meta.yml` file for local module `EXTRACT_FAMILY_REPS`
+  Added the default pipeline end-to-end nf-test.
```

Comment:
I think you borked this with a find/replace, it's now all on one line rather than bullets?